src/ape/managers/chain.py

```diff
@@ -482,7 +482,14 @@ def restore(self, snapshot_id: Optional[SnapshotID] = None):
         self.provider.revert(snapshot_id)
         self.account_history.revert_to_block(self.blocks.height)
 
-    def mine(self, num_blocks: int = 1, timestamp: Optional[int] = None) -> None:
+    def mine(
+        self,
+        num_blocks: int = 1,
+        timestamp: Optional[int] = None,
+        deltatime: Optional[int] = None,
+    ) -> None:
         if timestamp:
             self.pending_timestamp = timestamp
+        elif deltatime:
```

Comment:
I think it would be confusing if the user passes in both and the method succeeds but silently ignored one of the parameters.

I think we raise a `ValueError` if both params are present at the same time

Reply:
first case should be `if timestamp and deltatime: raise`